dumpling-overview.md

```diff
@@ -0,0 +1,257 @@
+
+{{< copyable "sql" >}}
+
+```sql
+update mysql.tidb set VARIABLE_VALUE = '10m' where VARIABLE_NAME = 'tikv_gc_life_time';
+```
+
+Finally, all the exported data can be imported back to TiDB using [Lightning](/tidb-lightning/tidb-lightning-tidb-backend.md).
+
+## Option list of Dumpling
+
+| Options | Usage | Default value |
+| --------| --- | --- |
+| -V or --version | Output the Dumpling version and exit directly |
+| -B or --database | Export specified databases |
+| -T or --tables-list | Export specified tables |
+| -f or --filter | Export tables that match the filter pattern. For the filter syntax, see[table-filter](/table-filter.md). | `"\*.\*"` (export all databases or tables) |
+| --case-sensitive | whether table-filter is case-sensitive | false (case-insensitive) |
+| -h or --host| The address of the linked node | "127.0.0.1" |
+| -t or --threads | The number of concurrent backup threads | 4 |
+| -r or --rows | Divide the table into specified rows of data (generally applicable for concurrent operations of splitting a large table into multiple files. |
+| -L or --logfile | Log output address. If it is empty, the log will be output to the console | "" |
+| --loglevel | Log level {debug,info,warn,error,dpanic,panic,fatal} | "info" |
+| --logfmt | Log output format {text,json} | "text" |
+| -d or --no-data | Do not export data (suitable for scenarios where only the schema is exported) |
+| --no-header | Export CSV files of the tables without generating header |
+| -W or --no-views| Do not export the views | true |
+| -m or --no-schemas | Do not export the schema with only the data exported |
+| -s or--statement-size | Control the size of the `INSERT` statements; the unit is bytes |
+| -F or --filesize | The file size of the divided tables. The unit must be specified such as `128B`, `64KiB`, `32MiB`, and `1.5GiB`. |
+| --filetype| Exported file type (csv/sql) | "sql" |
+| -o or --output | Exported file path | "./export-${time}" |
+| -S or --sql | Export data according to the specified SQL statement. This command does not support concurrent export. |
+| --consistency | flush: use FTWRL before the dump <br/> snapshot: specify the dump files' location through TSO <br/> lock: execute `lock tables read` on all tables to be dumped <br/> none: dump without adding locks, which sacrifices consistency <br/> auto: MySQL flush, TiDB snapshot | "auto" |
+| --snapshot | snapshot TSO; valid only when `consistency=snapshot` |
+| --where | Specify the scope of the table backup through the `where` condition |
+| -p or --password | The password of the linked node |
+| -P or --port | The port of the linked node | 4000 |
+| -u or --user | The username of the linked node | "root" |
+| --dump-empty-database | Export the `CREATE DATABASE` statements of the empty databases | true |
+| --tidbMemQuotaQuery | Maximum memory used by a single query when exporting TiDB database | 34359738368(32GB) |
```

Comment:
Invalid. The line is duplicated with the last line. Same deletion should be done in chinese version.